src/main/java/me/botsko/prism/database/sql/SqlSelectQueryBuilder.java

```diff
@@ -655,8 +662,7 @@ public QueryResult executeSelect(TimeTaken eventTimer) {
                     actions.add(baseHandler);
 
                 } catch (final SQLException e) {
-                    Prism.warn("Ignoring data from record #" + rowId + " because it caused an error:");
-                    e.printStackTrace();
+                    Prism.warn("Ignoring data from record #" + rowId + " because it caused an error:" , e);
```

Comment:
Codacy found an issue: [',' is preceded with whitespace.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)